### PR TITLE
[dagster-dlift] project scaffold

### PR DESCRIPTION
**Pull request by dpeng817**
## Summary & Motivation
Initial project scaffold for dagster-dlift.
## How I Tested These Changes
Added a single test for module load.

### CODE REVIEW

**Comment by dpeng817** [comment]
* **#25444** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25444?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25443** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25443?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dpeng817 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py [hunk 1]
```diff
@@ -381,6 +381,10 @@ def k8s_extra_cmds(version: str, _) -> List[str]:
         "examples/experimental/dagster-airlift/examples/kitchen-sink",
         always_run_if=has_dagster_airlift_changes,
     ),
+    PackageSpec(
+        "examples/experimental/dagster-dlift",
+        name=":dbt: Dbt Cloud-Lift",
```
Comment (benpankow) [inline .buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py:386]: this one does not have the same ring to it as airlift ngl

File: examples/experimental/dagster-dlift/setup.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    author="Dagster Labs",
+    author_email="hello@dagsterlabs.com",
+    license="Apache-2.0",
+    description="Tooling to assist with migrating from Airflow to Dagster.",
+    long_description=Path("README.md").read_text(),
+    long_description_content_type="text/markdown",
+    url=(
+        "https://github.com/dagster-io/dagster/tree/master/python_modules/libraries/"
+        "dagster-dlift"
+    ),
```
[the remark this answers is not in the document]
Comment (benpankow) [reply examples/experimental/dagster-dlift/setup.py:31]: The AI kind of cooked with this one

File: examples/experimental/dagster-dlift/setup.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        exec(fp.read(), version)
+    return version["__version__"]
+
+
+ver = get_version()
+pin = "" if ver == "1!0+dev" else NON_EDITABLE_INSTALL_DAGSTER_PIN
+CLI_REQUIREMENTS = ["click", "structlog"]
+setup(
+    name="dagster-dlift",
+    version="0.0.26",
```
Comment (benpankow) [inline examples/experimental/dagster-dlift/setup.py:21]:
```suggestion
    version=ver,
```

**Review by benpankow — APPROVED** [review]

File: .buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py (hunk 1 above)
Comment (dpeng817) [reply .buildkite/dagster-buildkite/dagster_buildkite/steps/packages.py:386]: yea...

**Pull request merged**